src/main/scala/com/greencatsoft/angularjs/AngularExecutionContext.scala

```diff
@@ -20,7 +20,7 @@ class AngularExecutionContext(
       case t: Throwable => reportFailure(t)
     }
 
-    timeout(run _, 0)
```

Comment:
Probably, we can just use `() => run()` instead? Or maybe we can specify the return type of the `run` method, by changing it to `def run(): Unit = try {...` or `def run() {...`. Please let me know how you think. Thanks!